2420/2/options_file.rb

```diff
@@ -0,0 +1,61 @@
+require 'optparse'
+require './versus.rb'
+require 'terminal-table'
+
+options = { name: '', top_bad_words: 0, top_words: 30 }
+OptionParser.new do |opts|
+  opts.banner = 'Usage: example.rb [options]'
+
+  opts.on('--top-bad-words [NUMBER]', Integer, 'Максимальное количество самых нецензурных участников') do |number|
+    options[:top_bad_words] = number
+  end
+  opts.on('--name [NAME]', String, 'Имя участника') do |name|
+    options[:name] = name
+  end
+  opts.on('--top-words [NUMBER]', Integer, 'Любимые слова человека') do |number|
+    options[:top_words] = number || 30
+  end
+end.parse!
+
+if options[:top_bad_words] > 0
+  battlers = Battler.battler_names_list
```

Comment:
add this to CONST NAMES and then use here and in the lines 42 and 58

you should get smth like this:

```
battlers NAMES.map { ... }
.sort_by...
```